deepchem/feat/deepvariant_pileup_featurizer.py

```diff
@@ -28,10 +28,11 @@ class PileupFeaturizer(Featurizer):
     >>> from deepchem.feat import RealignerFeaturizer, PileupFeaturizer
     >>> bamfile_path = 'deepchem/data/tests/example.bam'
     >>> reference_path = 'deepchem/data/tests/sample.fa'
-    >>> realigner= RealignerFeaturizer()
-    >>> windows_haplotypes = realigner.featurize((bamfile_path,reference_path))
+    >>> realign = RealignerFeaturizer()
+    >>> datapoint = (bamfile_path, reference_path, 299, 299, 6)
+    >>> windows_haplotypes = realign._featurize(datapoint)
     >>> pileup_feat = PileupFeaturizer()
-    >>> features = pileup_feat.featurize((windows_haplotypes, reference_path))
+    >>> features = pileup_feat._featurize((windows_haplotypes, reference_path))
```

Comment:
Don't use _featurize in a public docstring (it is intended to be a private method)